samples/dashboards/demo/live-weather/demo.js

```diff
@@ -584,6 +591,17 @@ async function setupDashboard() {
                             marker: {
                                 enabled: true,
                                 symbol: 'circle'
+                            },
+                            tooltip: {
+                                pointFormatter: function () {
+                                    return this.y + ' ' + activeParam.unit;
+                                }
+                            }
+                        },
+                        windbarb: {
+                            tooltip: {
+                                pointFormat: '<b>{point.value} m/s</b> ({point.beaufort})<br/>Wind direction: {point.direction} degrees',
+                                pointFormatter: null
```

Comment:
`pointFormat` should take precedence, so no need to set `pointFormatter` to null.

Reply:
When not setting pointFormat to null, this is what appears:
![image](https://github.com/highcharts/highcharts/assets/3593847/ecfc3860-dd96-407e-943a-44064e3c402b)

Reply:
Okay 👍